front_end/html/edit_exercise.html

```diff
@@ -35,6 +35,71 @@ <h2>Create exercise</h2>
 
     <!-- Global JavaScript -->
     <script>
+        // object for matching example code with back_end
```

Comment:
It seemed to me that loading in a dict of instructions/example code from webserver.py would slow things down as we'd have to make a request every time an instructor changed the back end. However, it also seems clunky to keep here inline with the rest of the script. Maybe we could store it in a separate js file?